betago/model.py

```diff
@@ -57,7 +65,7 @@ def next_move():
             json_result = jsonify(**result)
             return json_result
 
-        self.app.run(host='0.0.0.0', debug=True, threaded=True, use_reloader=False)
+        self.app.run(host='0.0.0.0', port=int("8081"), debug=True, threaded=True, use_reloader=False)
```

Comment:
I think it makes the most sense to use port `:8080`. It does not require special privileges and should work anywhere. Port `:8080` has traditionally been used as an alternate port for HTTP (and for apache servers, etc.) and is often redirected to port `:80` externally (so you can just visit the server root, no need for any port numbers). If a significant need arises, `argparse` can be used to check for any command line flags that override the default `:8080`. However, I don't think that it needs to be a priority.